test/module/transport-builtin-test.js

```diff
@@ -0,0 +1,246 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+const assert = require('assert');
+const napa = require('../../lib/index');
+let zoneKey = 'transport-test-zone';
+let zone = napa.zone.create(zoneKey, { workers: 4 });
+
+/// Construct an expected result string.
+/// constructExpectedResult(5, 5, 255) returns '0,0,0,0,0'
+/// constructExpectedResult(2, 5, 255) returns '0,0,255,255,255'
+/// constructExpectedResult(0, 5, 255) returns '255,255,255,255,255'
+global.constructExpectedResult =
+function constructExpectedResult(i, length, value) {
```

Comment:
do you mean padding with default value?

Reply:
yes, maybe a name like constructExpectedResult(currentIndex, length, defaultValue, value) will make it clearer?

---
In reply to: [155906961](https://github.com/Microsoft/napajs/pull/137#discussion_r155906961) [](ancestors = 155906961)